3p/iframe-messaging-client.js

```diff
@@ -25,6 +25,8 @@ export class IframeMessagingClient {
   constructor(win) {
     /** @private {!Window} */
     this.win_ = win;
+    /** @private {?string} */
+    this.rtvVersion_ = (win.AMP_CONFIG && win.AMP_CONFIG.v) || null;
```

Comment:
The script URL will eventually get written to the HTML head. So there's possibility. But I would think gpt library should always santinize .

But anyway, probably not a bad idea to do some format check here. It's just we introduced one more dependency to care about when we later want to change the RTV format.